src/components/entry/EntryOverview.js

```diff
@@ -137,7 +117,12 @@ const EntryOverview = ({ locationData, className }) => {
 
             <IconBesideText bold onClick={handleAddressClick} tabIndex={0}>
               <Map color={theme.secondaryText} size={20} />
-              <LocationText>{address}</LocationText>
+              <LocationText>
+                {locationData.address ??
```

Comment:
This is out of scope, but I'd prefer deconstructing all the attributes we need from `locationData` upfront.